src/main/java/pl/damianszczepanik/jenkins/buildhistorymanager/descriptors/actions/DeleteArtifactsWithPatternActionDescriptor.java

```diff
@@ -0,0 +1,50 @@
+package pl.damianszczepanik.jenkins.buildhistorymanager.descriptors.actions;
+
+import java.util.logging.Logger;
+
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import hudson.Extension;
+import hudson.model.Descriptor;
+import pl.damianszczepanik.jenkins.buildhistorymanager.model.actions.Action;
+import pl.damianszczepanik.jenkins.buildhistorymanager.model.actions.DeleteArtifactsWithPatternAction;
+
+/**
+ * Descriptor implementation needed to render UI for {@link DeleteArtifactsWithPatternAction}.
+ */
+@Extension
+@Symbol("DeleteArtifactsWithPattern")
+public class DeleteArtifactsWithPatternActionDescriptor extends Descriptor<Action>{
+    private static final Logger LOGGER = Logger.getLogger(DeleteArtifactsWithPatternActionDescriptor.class.getName());
```

Comment:
I believe VS Code is not best tool for development but never the less looks like it does not warn you that this LOGGER is not used variable

Reply:
Line 19 (LOGGER) has been removed.